### ISSUE TITLE
[Feature request] possible improvements 

### ISSUE THREAD

**Issue by KenosisM** [comfyanonymous/ComfyUI#146]
1. allow to use del to kill modules (could it be just my keyboard malfunctioning??)
2. redesign the conditioning area module, or make a new type:

![QQ图片20230318164223](https://user-images.githubusercontent.com/15232782/226095299-d2acf786-3ed8-43ba-a767-1f456790b25a.png)

that would allow users to directly draw their separated areas on canvas instead of cauculate all the coordinates. Since all modules are recorded size/position in the json, guess the editor should have all the data needed.

3.Add more precise controlnet controls: resize the input image, decide which steps to activate controlnet, midas resolution, etc, what webui has. And preprocessor support.
4. give controlnetapply built-in controlnet loader and image loader, or make a "simple controlnet apply" and the same to t2i networks.
5. allow users to save certain modules and their connections as custom modules

**Comment by KenosisM** [comment]
Also should there be a stop processing button? Or this feature is already done?

**Comment by ikcikoR** [comment]
@KenosisM Yeah you can view queue of processes on the right if you press the button and stop running ones from there
